db/migrate/20250109050737_change_denial_reason_description_to_denial_reason_in_claim_denied_claim_reasons.rb

```
@@ -0,0 +1,5 @@
+class ChangeDenialReasonDescriptionToDenialReasonInClaimDeniedClaimReasons < ActiveRecord::Migration[7.0]
```

rubocop (1.69.1)

[convention] Missing top-level documentation comment for `class ChangeDenialReasonDescriptionToDenialReasonInClaimDeniedClaimReasons`.

(Style/Documentation)